install/ui/src/freeipa/association.js

```diff
@@ -822,12 +822,37 @@ IPA.association_table_field = function (spec) {
 
     spec = spec || {};
 
+    /**
+     * Turn off decision whether the field is writable according to metadata.
+     * The source of rights will be only ACLs.
+     *
+     * @property {Boolean}
+     */
+    spec.check_writable_from_metadata = spec.check_writable_from_metadata === undefined ?
+                        false : spec.check_writable_from_metadata;
+
     var that = IPA.field(spec);
 
+    /**
+     * In case that there is set acl_param in spec then this property will be
+     * set to true and it checks ACLs.
+     */
+    that.acl_param_set = !!spec.acl_param;
+
     that.load = function(data) {
         that.values = that.adapter.load(data);
         that.widget.update(that.values);
         that.widget.unselect_all();
+
+        if (that.acl_param_set) {
+            var record = that.adapter.get_record(data);
+            that.load_writable(record);
+            that.handle_acl();
+        }
+    };
+
+    that.handle_acl = function() {
+        if (!that.writable) that.widget.set_enabled(false);
```

Comment:
Yes, you are right Association_table_field does not use FieldBinder and even in its load method uses that.widget.update method.